dev_guide/app_tutorials/maven_tutorial.adoc

```diff
@@ -86,15 +93,18 @@ $ oc set probe dc/nexus \
 ====
 If you do not want persistent storage, continue to
 xref:nexus-connecting-to-nexus[Connecting to Nexus]. However, your cached
-dependencies will be lost if the pod is restarted for any reason.
+dependencies and any configuration customization will be lost if the pod is
+restarted for any reason.
 ====
 
 Create a persistent volume claim (PVC) for Nexus, so that your dependencies
-will not be lost when the pod running the server terminates. If you do not have
-administrator access on your cluster, ask your system administrator to create a
-Read/Write Persistent Volume for you.  Otherwise, read
-xref:../volumes.adoc#dev-guide-volumes[Managing Volumes in {product-title}] for instructions on
-creating a persistent volume.
+will not be lost when the pod running the server terminates. PVCs require
+available persistent volumes (PV) in your cluster. If there are no PVs
+available in your cluster and you do not have administrator access on your
+cluster, ask your system administrator to create a Read/Write Persistent Volume
+for you.  Otherwise, read
+xref:../volumes.adoc#dev-guide-volumes[Managing Volumes in {product-title}] for
+instructions on creating a persistent volume.
 
 Add a PVC to the Nexus deployment configuration.
```

Comment:
This is what I see. Maybe I don't have the latest?

```
$ oc set volume --help
Update volumes on a pod template

This command can add, update or remove volumes from containers for any object
that has a pod template (deployment configs, replication controllers, or pods).
You can list volumes in pod or any object that has a pod template. You can
specify a single object or multiple, and alter volumes on all containers or
just those that match a given name.

If you alter a volume setting on a deployment config, a deployment will be
triggered. Changing a replication controller will not affect running pods, and
you cannot change a pod's volumes once it has been created.

Volume types include:

* emptydir (empty directory) *default*
    A directory allocated when the pod is created on a local host, is removed when
    the pod is deleted and is not copied across servers
* hostdir (host directory)
    A directory with specific path on any host (requires elevated privileges)
* persistentvolumeclaim or pvc (persistent volume claim)
    Link the volume directory in the container to a persistent volume claim you have
    allocated by name - a persistent volume claim is a request to allocate storage.
    Note that if your claim hasn't been bound, your pods will not start.
* secret (mounted secret)
    Secret volumes mount a named secret to the provided directory.

For descriptions on other volume types, see https://docs.openshift.com

Usage:
  oc set volumes RESOURCE/NAME --add|--remove|--list [options]

Examples:
  # List volumes defined on all deployment configs in the current project
  oc set volume dc --all

  # Add a new empty dir volume to deployment config (dc) 'registry' mounted under
  # /var/lib/registry
  oc set volume dc/registry --add --mount-path=/var/lib/registry

  # Use an existing persistent volume claim (pvc) to overwrite an existing volume 'v1'
  oc set volume dc/registry --add --name=v1 -t pvc --claim-name=pvc1 --overwrite

  # Remove volume 'v1' from deployment config 'registry'
  oc set volume dc/registry --remove --name=v1

  # Create a new persistent volume claim that overwrites an existing volume 'v1'
  oc set volume dc/registry --add --name=v1 -t pvc --claim-size=1G --overwrite

  # Change the mount point for volume 'v1' to /data
  oc set volume dc/registry --add --name=v1 -m /data --overwrite

  # Modify the deployment config by removing volume mount "v1" from container "c1"
  # (and by removing the volume "v1" if no other containers have volume mounts that reference it)
  oc set volume dc/registry --remove --name=v1 --containers=c1

  # Add new volume based on a more complex volume source (Git repo, AWS EBS, GCE PD,
  # Ceph, Gluster, NFS, ISCSI, ...)
  oc set volume dc/registry --add -m /repo --source=<json-string>

Options:
      --add=false: Add volume and/or volume mounts for containers
      --all=false: select all resources in the namespace of the specified resource types
      --claim-mode='ReadWriteOnce': Set the access mode of the claim to be created. Valid values are ReadWriteOnce (rwo), ReadWriteMany (rwm), or ReadOnlyMany (rom)
      --claim-name='': Persistent volume claim name. Must be provided for persistentVolumeClaim volume type
      --claim-size='': If specified along with a persistent volume type, create a new claim with the given size in bytes. Accepts SI notation: 10, 10G, 10Gi
      --configmap-name='': Name of the persisted config map. Must be provided for configmap volume type
      --confirm=false: Confirm that you really want to remove multiple volumes
  -c, --containers='*': The names of containers in the selected pod templates to change - may use wildcards
  -f, --filename=[]: Filename, directory, or URL to file to use to edit the resource.
      --list=false: List volumes and volume mounts for containers
  -m, --mount-path='': Mount path inside the container. Optional param for --add or --remove
      --name='': Name of the volume. If empty, auto generated for add operation
  -o, --output='': Display the changed objects instead of updating them. One of: json|yaml
      --output-version='': Output the changed objects with the given version (default api-version).
      --overwrite=false: If true, replace existing volume source and/or volume mount for the given resource
      --path='': Host path. Must be provided for hostPath volume type
      --remove=false: Remove volume and/or volume mounts for containers
      --secret-name='': Name of the persisted secret. Must be provided for secret volume type
  -l, --selector='': Selector (label query) to filter on
      --source='': Details of volume source as json string. This can be used if the required volume type is not supported by --type option. (e.g.: '{"gitRepo": {"repository": <git-url>, "revision": <commit-hash>}}')
  -t, --type='': Type of the volume source for add operation. Supported options: emptyDir, hostPath, secret, configmap, persistentVolumeClaim

Use "oc options" for a list of global command-line options (applies to all commands).
```